R/ConceptSets.R

```diff
@@ -237,10 +237,10 @@ runConceptSetDiagnostics <- function(connection,
   
   conceptSets <- compareAndCombineConceptSetsDefinitionForCohorts(subset)
   uniqueConceptSets <- conceptSets %>% 
-                        dplyr::select(.data$uniqueConceptSetId, .data$conceptSetName, .data$conceptSetSql) %>% 
-                        dplyr::distinct()
-  instantiateUniqueConceptSets(cohorts = subset,
```

Comment:
I also removed the `cohorts = subset` parameter here since this appears to have been removed and was causing an error.